src/frontend/parser.messages

```diff
@@ -2,7 +2,7 @@ program: WHILE
 ##
 ## Concrete syntax: while
 ##
-## Ends in an error in state: 599.
+## Ends in an error in state: 609.
```

Comment:
Definitely not something for this PR but these state number changes make reviewing diffs pretty annoying. And the numbers don't really matter anyway, so maybe we could just write a pre-commit script that deletes these comments.

Reply:
I find them useful because they’re printed if a parse error occurs when using `—debug-parse` and I can use them to quickly jump to errors (especially if there are duplicate messages or if I’m writing errors for new states in the parser)

Reply:
I believe some variation of `add_missing_messages.py` script can regenerate them on demand but I forget how all that works. If that would work then I'd like to keep the numbers out of git history.